### ISSUE TITLE
Please consider uploading gaiasandbox to Debian

### ISSUE THREAD

**Issue by olebole** [langurmonkey/gaiasky#39]
Hi,

I am writing you on the behalf of the Debian Astronomy Team. Since you already created a Debian package: how about uploading it to Debian? Then it would be automatically part of the distribution (as well as on Ubuntu, Mint etc.). We could help you getting through all the formalities needed to upload it.

If you are interested, please contact me either personally, or via our [mailing list](https://lists.debian.org/debian-astro/).

Best regards

Ole

**Comment by langurmonkey** [comment]
Thanks for the suggestion!
Yes, I thought about that, but I feel the GS is still too immature to upload it to external repositories. I will probably look into this when we hit version 1 and the code base is stable enough.

**Comment by olebole** [comment]
Hmm, does that mean that you are going to upload the 1.0 version, or did you decide that it is not worth at all? To me. gaiasandbox still looks like a good candidate.

**Comment by langurmonkey** [comment]
Yes, it is now the time to do it and I was actually starting to look into this. So what would be required to make an official submission?

**Comment by olebole** [comment]
Ah, OK; glad to hear :-)
"In principle"  it is simple:  The package must follow the [Debian Policy](https://www.debian.org/doc/debian-policy/), and you need a sponsor to get it uploaded (where I would volunteer).

I wrote a tutorial for [how to create a Debian package](https://wiki.debian.org/DebianAstro/AstropyPackagingTutorial/Preparation). Although it is focused on astropy affiliated packages, the first (more or less formal) steps are the same. Can you check this out?

**Comment by hakphar** [comment]
Any movement on this? I'm not associated with the above, but I'd really like to see this happen.

**Comment by olebole** [comment]
... and I am still happy to assist.

**Comment by langurmonkey** [comment]
It's on the TODO list, it should happen *soon™*, hopefully. In the meantime, you can use the `.deb` file we provide with each release. It should install without hassle.

**Comment by olebole** [comment]
... and I would still be happy to assist.
@hakphar This does not nexessarily need to be done by the original authors; anyone can do this. So if you are willing to do this yourself (or together with @langurmonkey), this would work as well.

**Comment by hakphar** [comment]
Nice. I've got some free time this next week, I'll get cracking on it.

**Comment by langurmonkey** [comment]
Sure, we can work it out together ;)

**Comment by olebole** [comment]
That's the best ow all worlds :-)

**Comment by hakphar** [comment]
:D

**Comment by hakphar** [comment]
Just a ping to let you know that I've created an account on alioth(this happened outside of this issue, so I'm a step ahead.) And will send the bugwish some time tomorrow, then get to the actual packaging.

**Comment by hakphar** [comment]
Working on the wording of the bugreport, anything you want/need to be in the wishlist, @langurmonkey ? I grabbed a desc from the Gaia Win10 System tray app, but I want to make sure I keep in line with your vision when I describe the whole thing.

**Comment by langurmonkey** [comment]
I would use the same description and values as in the [AUR PKGBUILD](https://github.com/langurmonkey/gaiasky/blob/master/desktop/installerscripts/arch/PKGBUILD) file. Thanks for your work, btw!

**Comment by olebole** [comment]
Driven by the [announcement yesterday](http://sci.esa.int/gaia/60192-gaia-creates-richest-star-map-of-our-galaxy-and-beyond/): would you still like to get this uploaded?

**Comment by langurmonkey** [comment]
Yes, would still be nice, but I won't be able to look at this for a couple of weeks at least, and I guess @hakphar just gave up...
